codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -49,18 +49,21 @@
     void writePendingBytes() throws Http2Exception;
 
     /**
-     * Set the active listener on the flow-controller.
-     *
-     * @param listener to notify when the a write occurs, can be {@code null}.
+     * Determine if the {@code stream} has bytes remaining for use in the flow control window.
+     * <p>
+     * Note that this only takes into account HTTP/2 flow control. It does <strong>not</strong> take into account
+     * the underlying {@link io.netty.channel.Channel#isWritable()}.
+     * @param stream The stream to test.
+     * @return {@code true} if if the {@code stream} has bytes remaining for use in the flow control window.
+     * {@code false} otherwise.
      */
-    void listener(Listener listener);
```

Comment:
@nmittler - Sure, but is this the last comment? I added a bunch of stuff to propagate user exceptions into GO_AWAY frames (instead of just logging them and having docs say it is a programming error to throw).